files/system/usr/share/secureblue/templates/.bash_profile

```diff
@@ -0,0 +1,8 @@
+# .bash_profile
```

Comment:
Placing the templates under `etc/skel` will solve https://github.com/secureblue/secureblue/issues/448.